README.txt

```diff
@@ -6,3 +6,27 @@ Please see the LLVM testing infrastructure guide:
   https://llvm.org/docs/TestSuiteGuide.html
 
 for more information on the contents of this repository.
+
```

Comment:
I suggest using Readme.md here, will it prevail over Readme.txt in github visualization?
In particular links to License and Contributing are broken below.

Reply:
yes. Will restore original README.txt and add README.md specific for our project.